spread.yaml

```diff
@@ -12,7 +12,7 @@ environment:
     PROJECT_PATH: $GOHOME/src/github.com/snapcore/snapd
     # /usr/lib/go-1.6/bin for trusty (needs to be last as we use
     # a different go in gccgo tests)
```

Comment:
If the change below is correct (I don't know, I'll let other people say :-) ), then this comment should probably be removed.

Reply:
Oopsi, once again you have a sight of an eagle :-). My bad, I had to remove it from the comment. Thank you.

Reply:
@are-scenic I didn't see this at first, but yes you should split this change to spread.yaml to a separate PR, then we can add the label to this PR to skip spread tests

Reply:
The commit responsible for the change has been removed.